src/plugins/chart_expressions/expression_gauge/public/components/gauge_component.tsx

```diff
@@ -168,272 +163,250 @@ function getTicks(
-            onRenderChange={onRenderChange}
-            locale={i18n.getLocale()}
-            {...getOverridesFor(overrides, 'settings')}
-          />
-          <Goal
-            id="goal"
-            subtype={getSubtypeByGaugeType(gaugeType)}
-            base={bands[0]}
-            target={goal && goal >= bands[0] && goal <= bands[bands.length - 1] ? goal : undefined}
-            actual={actualValue}
-            tickValueFormatter={({ value: tickValue }) => tickFormatter.convert(tickValue)}
-            tooltipValueFormatter={(tooltipValue) => tickFormatter.convert(tooltipValue)}
-            bands={bands}
-            ticks={ticks}
-            domain={{ min, max }}
-            bandFillColor={
-              colorMode === GaugeColorModes.PALETTE
-                ? (val) => {
-                    const value = getPreviousSectionValue(val.value, bands);
-
-                    const overridedColor = overrideColor(
-                      value,
-                      args.percentageMode ? bands : args.palette?.params?.stops ?? [],
-                      args.percentageMode ? tickFormatter : undefined
-                    );
-
-                    if (overridedColor) {
-                      return overridedColor;
-                    }
-                    return args.palette
-                      ? getColor(value, args.palette, bands, args.percentageMode) ?? TRANSPARENT
-                      : TRANSPARENT;
-                  }
-                : () => TRANSPARENT
-            }
-            labelMajor={labelMajorTitle ? `${labelMajorTitle}${majorExtraSpaces}` : labelMajorTitle}
-            labelMinor={labelMinor ? `${labelMinor}${minorExtraSpaces}` : ''}
-            {...extraTitles}
-            {...goalConfig}
-            {...getOverridesFor(overrides, 'gauge')}
-          />
-        </Chart>
-        {commonLabel && <div className="gauge__label">{commonLabel}</div>}
-      </div>
-    );
-  }
-);
+    return args.palette
+      ? getColor(value, args.palette, bands, args.percentageMode) ?? TRANSPARENT
+      : TRANSPARENT;
+  };
+  const getColorConfig = (): BulletProps['colorBands'] => {
+    if (colorMode === GaugeColorModes.NONE)
+      return [euiTheme.colors.lightShade, euiTheme.colors.lightShade];
+
+    return {
+      steps: bands,
+      colors: bands.map((b) => colorFn(b)).slice(1),
+    };
+  };
+
+  return (
+    <div className="gauge__wrapper">
+      <Chart {...getOverridesFor(overrides, 'chart')}>
+        <Settings
+          noResults={<EmptyPlaceholder icon={icon} renderComplete={onRenderChange} />}
+          debugState={window._echDebugStateFlag ?? false}
+          theme={[{ background: { color: 'transparent' } }]}
+          baseTheme={chartBaseTheme}
+          ariaLabel={args.ariaLabel}
+          ariaUseDefaultSummary={!args.ariaLabel}
+          onRenderChange={onRenderChange}
+          locale={i18n.getLocale()}
+          {...getOverridesFor(overrides, 'settings')}
+        />
+        <Bullet
```

Comment:
I think you also need to wire up the `valueLabels` to the dimension names so that the tooltip can pick these values and use them instead of the default ones.

<img width="1093" alt="Screenshot 2024-03-13 at 09 38 24" src="https://github.com/elastic/kibana/assets/1421091/04b81025-f923-444b-ad02-4dee2ccbad69">

We probably also need to replace the default one to not be `Static value: 10` but something like min,max, goal

Reply:
Yeah we can wire up the `valueLabels` but I'm not sure what the `Active` and `Value` should be, they both represent the `Metric` but in different ways.

Secondly, what are you referring to for the `Static value: 10`? Are you saying we should change `Suggested value: 0` to `Static value: 0`?

![Lens - Elastic 2024-03-13 at 6 06 39 PM](https://github.com/elastic/kibana/assets/19007109/246c06a8-4dd3-4670-93b3-0e88e269342e)

Reply:
I think you can use `Metric` or it's custom name instead of the`Value` and you can use a label like `Current` to specify the `current hovered chart value`. wdyt?

Reply:
Let's track this conversation in https://github.com/elastic/kibana/issues/152515
For now let's keep it like that and we can fix it later

Reply:
Updated simple naming in 8272065f7b64d1f6ac007864b58f717448fa497a